### ISSUE TITLE
Replace ```10x``` with actual assay type

### ISSUE THREAD

**Issue by andreasbueckle** [hubmapconsortium/ccf-ui#1204]
In EUI, replace current ```assay_type``` ```10x``` with actual assay type such as ```sc-rna-seq```

**Comment by bherr2** [comment]
We'll need to discuss this more. If we switch to passing through assay types (which ones, at what level, do we just let the provider say and we hope for the best?) there will likely be an explosion of "technologies" and the EUI filter will break.

**Comment by danqaurooni** [comment]
Yes to discussing this further and coming up with a controlled vocabulary for technologies, among other things. I am in the process of updating the RUI SOPs and this is one thing we're lacking. Once we do, I also need to go over existing RUI collections and apply that controlled vocabulary.
